community/server/src/main/java/org/neo4j/server/Bootstrapper.java

```diff
@@ -76,6 +77,7 @@ public static void main( String[] args )
     public int start( File configFile, Pair<String, String> ... configOverrides )
     {
         LogProvider userLogProvider = FormattedLogProvider.withoutRenderingContext().toOutputStream( System.out );
+        System.setOut( NullPrintStream.getInstance() );
```

Comment:
Actually, standard practice has always been to redirect to close the original stdout and then re-open it pointing to /dev/null, for exactly the reason that you don't want things to blow up if they're writing to stdout internally. If you can describe a problem that we'd be hiding in this way, that would be interesting to look into.